NuGet.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
meta-comment: I'm just trusting you on what should be `.gitignore`d and what shouldn't be...